src/system_monitor.c

```diff
@@ -28,10 +30,14 @@
 #include "manage.h"
 #include "simulator.h"
 #include "system_monitor.h"
+#include "position.h"
 
 #define WATCHDOG_HARDWARE_PERIOD_MS 30000  /* Actual period may vary +88% -32% */
 #define WATCHDOG_THREAD_PERIOD_MS 15000
 
+/* Distance in meters between rover and surveyed base station position */
```

Comment:
Isn't this the distance between the surveyed and calculated base station position? In which case something like
```
/* Maximum distance between calculated and surveyed base station single point position for error checking. */
```